google/resource_bigquery_dataset.go

```diff
@@ -180,6 +226,45 @@ func resourceDataset(d *schema.ResourceData, meta interface{}) (*bigquery.Datase
 		dataset.Labels = labels
 	}
 
+	if v, ok := d.GetOk("access"); ok {
+		access := []*bigquery.DatasetAccess{}
+		for _, m := range v.([]interface{}) {
+			da := bigquery.DatasetAccess{}
+			msi := m.(map[string]interface{})
```

Comment:
does msi just stand for map[string]interface? if so maybe a sliiiightly more descriptive var name could be used